docs/endpoint.md

```diff
@@ -0,0 +1,106 @@
+# Endpoint
+
+Each request begins and ends its lifecycle inside your application in an endpoint. The endpoints handles starting the server and transforming request passing it through several defined plugs before calling the [Router](routing.html).
+
```

Comment:
`Starting the web server and transforming requests through several defined plugs...`